uwblueprint/sistering
# Add Chakra UI component library to frontend

## Ticket link

N/A - this a prerequisite task for some upcoming frontend work.



## Implementation description
* Followed Chakra's [installation guide](https://chakra-ui.com/docs/getting-started)
* Defined theme according to our [design system](https://www.figma.com/file/6zTSxaydHqW9YDmxr4wF1f/Sistering-Design-System)
* Made minor modifications to existing starter-code components to improve display



## Steps to test
1. Run the application
```
docker-compose up --build
```
2. Navigate to the frontend and login: `http://localhost:3000`
3. Check the design system on the default page
![image](https://user-images.githubusercontent.com/30205929/144728668-fb004633-e294-437a-91fa-6ddeb8ccbd92.png)



## What should reviewers focus on?
* Chakra best practices for theming


## Checklist
